capsules/src/alarm.rs

```diff
@@ -100,6 +102,7 @@ impl<A: Alarm<'a>> Driver for AlarmDriver<'a, A> {
     /// - `2`: Read the the current clock value
     /// - `3`: Stop the alarm if it is outstanding
     /// - `4`: Set an alarm to fire at a given clock value `time`.
+    /// - `5`: Set an alarm to fire at a given clock value `time` relative to `bow`.
```

Comment:
s/bow/now

Reply:
Can we somehow note in the comments that this system is experimental? It's likely, given some suggested new timer interfaces that we may not retain this system call long term (indeed system call 4 will likely change as well), so we want to highlight the status it would have.

Reply:
I added experimental in the comments